### DISCUSSION TITLE
/ringing-your-doorbell-on-your-sonos-speakers/

### DISCUSSION THREAD

**Discussion by comment-migrator** [estruyf/eliostruyf-comments#480] [Comments]
Ringing your doorbell on your Sonos speakers
        
[https://www.eliostruyf.com/ringing-your-doorbell-on-your-sonos-speakers/](https://www.eliostruyf.com/ringing-your-doorbell-on-your-sonos-speakers/)

**Comment by n006** [comment]
Thank you very much!
Since Alexa is not supported anymore by IFTTT from the 1st of November, do you aleready have some tips to fill the gap?

Best
Paul
